### PR TITLE
Persistent Keylogger - Telegram Based

### PR DESCRIPTION
**Pull request by aleff-github**

### CODE REVIEW

File: payloads/library/execution/Persistent_Keylogger-Telegram_Based/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Persistent Keylogger - Telegram Based
+
+A script used to configure a persistent keylogger on a Linux computer trough a pre-configured Telegram Bot.
```
Comment (dallaswinger) [inline payloads/library/execution/Persistent_Keylogger-Telegram_Based/README.md:3]: through*

File: payloads/library/execution/Persistent_Keylogger-Telegram_Based/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**Category**: Execution
+
+## Dependencies
+
+* Internet Connection
+
+## Description
+
+A script used to configure a persistent keylogger on a Linux computer trough a pre-configured Telegram Bot.
```
Comment (dallaswinger) [inline payloads/library/execution/Persistent_Keylogger-Telegram_Based/README.md:13]: through*

**Comment by dallaswinger** [comment]
tiny typos, otherwise lgtm

File: payloads/library/execution/Persistent_Keylogger-Telegram_Based/README.md (hunk 1 above)
Comment (aleff-github) [reply payloads/library/execution/Persistent_Keylogger-Telegram_Based/README.md]: Solved > bda84a99b160f20c4e3ceae0f59b5b77eb112745

File: payloads/library/execution/Persistent_Keylogger-Telegram_Based/README.md (hunk 2 above)
Comment (aleff-github) [reply payloads/library/execution/Persistent_Keylogger-Telegram_Based/README.md]: Solved > bda84a99b160f20c4e3ceae0f59b5b77eb112745

**Comment by aleff-github** [comment]
> tiny typos, otherwise lgtm

Solved > bda84a99b160f20c4e3ceae0f59b5b77eb112745

**Pull request merged**